readme_template.md

```diff
@@ -255,6 +257,8 @@ Gentoo users may find [`sb-hosts`](https://github.com/PF4Public/gentoo-overlay/t
 
 ## Updating hosts file on Windows
 
+(NOTE: If you love the hosts files produced by this repository but are only interested in managing the hosts files you actually use on Windows and not actually utilizing the full functionality of the awesome script that is the core of this repository, please see the below `Third-Party Hosts Managers` section for hosts manager solutions, such as [Unified Hosts AutoUpdate](https://github.com/ScriptTiger/Unified-Hosts-AutoUpdate "Unified Hosts AutoUpdate").)
+
```

Comment:
@ScriptTiger this can be reduced to, "See also some [third-party Hosts managers], listed below.

Reply:
Done. This will be the second note to refer to that section, so any future issues regarding this will be reasonably exhaustive, I feel. That's why I write so much the first time just to give Windows users that little bit of extra help they may need.